backend/app/services/draft_service.py

```
@@ -0,0 +1,279 @@
+        if not self.member_repo.has_access(case_id, user_id):
+            raise PermissionError("You do not have access to this case")
+
+        # 2. Retrieve evidence metadata from DynamoDB
+        evidence_list = get_evidence_by_case(case_id)
+
+        # Check if there's any evidence
+        if not evidence_list:
+            raise ValidationError("사건에 증거가 하나도 없습니다. 증거를 업로드한 후 초안을 생성해 주세요.")
+
+        # Filter for completed evidence only (status="done")
+        completed_evidence = [
```

Ruff (0.14.5)

Local variable `completed_evidence` is assigned to but never used

Remove assignment to unused variable `completed_evidence`

(F841)